### DISCUSSION TITLE
Project creation in the Supabase dashboard is disabled for this Vercel-managed organization

### DISCUSSION THREAD

**Discussion by khakimio** [supabase/supabase#29216] [Questions]
After the partnership Vercel + Supabase, Vercel template stopped working

This template SaaS in 2024 - Next.js, Supabase & Stripe https://www.youtube.com/watch?v=I7CFD99sp1g&t=54s
 
 Errors:
<img width="1058" alt="Screenshot 2024-09-11 at 11 53 32" src="https://github.com/user-attachments/assets/6682f724-82ed-4966-b5d0-30835d1427ab">
<img width="1036" alt="Screenshot 2024-09-11 at 11 54 43" src="https://github.com/user-attachments/assets/c2dc9216-bd7a-4815-a298-66daddf037d4">

**Comment by satoshicancode** [comment]
I got the same error :(

**Reply by satoshicancode** [reply]
Hello @khakimio, do you know how to fix it?

**Reply by khakimio** [reply]
I manually dragged the folders into my project and configured the supabase + vercel bundle myself

**Reply by satoshicancode** [reply]
@khakimio Could you please guide me how to setup manual?

**Reply by ZaidGhaffar** [reply]
@khakimio  can you guide us please as well

**Reply by khakimio** [reply]
problem is not solved, I had to do it all manually
